### ISSUE TITLE
Loans are not totalling correctly 

### ISSUE THREAD

**Issue by LESupport** [lend-engine/lend-engine-app#98]
I've had reports of this from other admin too, none of the totals add up correctly. 

The totals on the dashboard and top right total on the loans page for borrow.libraryofstuff.co.uk are: 

Pending 2, actual 0
On Loan 13, actual 15
Reserved 28, actual 36

![loans](https://user-images.githubusercontent.com/76882506/112123274-07bd7700-8bb9-11eb-82ec-6b3fea747999.png)
![pending 0](https://user-images.githubusercontent.com/76882506/112123333-1441cf80-8bb9-11eb-9d33-047a9242f89b.png)
![on loan](https://user-images.githubusercontent.com/76882506/112123355-19068380-8bb9-11eb-804d-0e5c161a2af1.png)
![36 reserved](https://user-images.githubusercontent.com/76882506/112123365-1ad04700-8bb9-11eb-8ae4-a2873553c492.png)

**Comment by YvetteKlinkenberg** [comment]
I noticed this as well. I figured this had to do with the unique IDs vs total number of items within the reservation. In the example there are 3 entries with ID 1191, however they may be counted as 1 reservation in the dashboard. However, the pending loans in the example I cannot explain with this theory

**Comment by LESupport** [comment]
Yes, you are right, the total loans do add up correctly with a number of multiple item loans, that's good to know. I think Item loans would be better on the dashboard, but that might just be me. 

The pending loans for me have been "1" for months, in the last week or so it changed to "2" I've never had any pending loans that I know of.

**Comment by YvetteKlinkenberg** [comment]
I also prefer the number of items on the dashboard. I notice that some users book multiple items in 1 loan and other create separate loans (or book something, cancel a part of it and make a separate new loan for somethin else). It gives a better impression if you have the total number of items which are reserved.

It took me a while before I figured out the "pending" status, but for as far as I know these are Admin Check-outs for a member which are not completely processed. You need to click twice a "check-out" button before the item is set from available to "on loan" by this member. If you forget the 2nd check-out button, the item remains in the pending status. I don't use that status, so for me it's just an extra action before the member can take the items home.

**Comment by YvetteKlinkenberg** [comment]
![image](https://user-images.githubusercontent.com/17195572/129945910-72aecd40-81fc-4f02-9d71-16671141e5fe.png)
I cannot even explain the difference between the 6 reservations in the top corner and the total reservations in the system (note the large date range I took in the filter)
